doc/reference/networking/coap.rst

```diff
@@ -6,6 +6,177 @@ CoAP
 Overview
 ********
 
+The Constrained Application Protocol (CoAP) is a specialized web transfer
+protocol for use with constrained nodes and constrained (e.g., low-power,
+lossy) networks. It provides a convenient API for RESTful Web services
+that support CoAP's features. For more information about the protocol
+itself, see `IETF RFC7252 The Constrained Application Protocol <https://tools.ietf.org/html/rfc7252>`_.
+
+Zephyr provides a CoAP library which supports client and server roles.
+The library is configurable as per user needs. The Zephyr CoAP library
+is implemented using plain buffers. Users of the API has to create sockets
```

Comment:
Either change to ``Users of the API create sockets...`` or ``Users of the API have to create sockets ...`` (my preference is the first option).